src/main/java/org/opentripplanner/middleware/spark/Main.java

```diff
@@ -85,6 +91,9 @@ private static void initializeHttpEndpoints() throws IOException {
         // available at http://localhost:4567/async
         spark.get("/async", (req, res) -> BasicOtpDispatcher.executeRequestsAsync());
 
+        // available at http://localhost:4567/api/admin/bugsnag/eventsummary
+        spark.get(API_PREFIX + "admin/bugsnag/eventsummary", BugsnagController::getEventSummary);
```

Comment:
Sorry, I am just noticing this one while working on the API docs... could you move this to `BugsnagController.register` similar to what `LogController` does? Thanks.

Reply:
Updated.